### DISCUSSION TITLE
Can we search and export prompt by keyword?

### DISCUSSION THREAD

**Discussion by LinChiSin** [timqian/openprompt.co#23] [Q&A]

**Comment by timqian** [comment]
we support searching for starred prompt (For now, to encourage starring useful prompts, we don't support search all prompts, but only starred )
<img width="1290" alt="Screen Shot 2023-03-30 at 11 20 14" src="https://user-images.githubusercontent.com/5512552/228720201-44b7785f-1d43-4cc6-8f72-d1480eb1b61c.png">

What do you mean by `export prompt`?

**Reply by LinChiSin** [reply]
Thanks for your reply.

What I mean by "export prompt" is I'd like have a button to export those fasinating prompts to a json or something else, which could help in my native ChatGPT cilent (iOS/macOS app, for instance).

**Reply by timqian** [reply]
The top prompts can be found here: https://github.com/timqian/openprompt.co/blob/main/TopPrompts.json
It is synced every 24 hours. For now it contain top 20 prompts, as we get more and more high quality prompts, the list will get longer

**Comment by timqian** [comment]
As we get more useful prompts, the list has been expend to top 50
